comms/comms.go

```diff
@@ -38,8 +38,8 @@ type Package interface {
 	// (agent advertisement to advertise)
 	AdvertiseAgent(defs.AgentAdvert) error
 
-	// (map of assets:hashes, lock for asset map)
-	ListenForAgent(assetProvider) error
+	// (map of assets:hashes, base asset URL)
+	ListenForAgent(assetProvider, string) error
```

Comment:
True, and I think we already discussed this in a previous review (so that's on me). I opened https://github.com/toddproject/todd/issues/137 to do this in a future PR